.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
### Descriptive Summary Example :heavy_check_mark:
![hammas-summary](https://user-images.githubusercontent.com/93039359/176921727-17499afe-c29c-47f8-823e-a3cfc70feae1.png)

- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: